components/sup/src/command/shell.rs

```diff
@@ -15,13 +15,14 @@
 use std::env;
 use std::ffi::CString;
 use std::ptr;
+use std::path::PathBuf;
 
 use hcore::env as henv;
 use hcore::fs::find_command;
 use libc;
 
 use error::{Error, Result};
-use util::path::busybox_paths;
+use util::path::append_interpreter_and_path;
```

Comment:
Rust style tends to prefer not directly importing funtions, and import the module then module qualify the function.

https://aturon.github.io/style/imports.html#prefer-fully-importing-types/traits-while-module-qualifying-functions.